tests/unittests/OperatorTest.cpp

```diff
@@ -580,6 +580,114 @@ TEST_P(InterpAndCPU, batchedReduceMeanQuantizedWithAxis) {
   }
 }
 
+TEST_P(InterpAndCPU, batchedReduceMeanUsingAvgPool) {
+  std::vector<size_t> dims = {3, 20, 4, 8};
+
+  size_t dimResMul = dims[0] * dims[1];
+  size_t dimTmpMul = dimResMul * dims[2];
+  size_t dimAllMul = dimTmpMul * dims[3];
+
+  auto *batch = mod_.createPlaceholder(ElemKind::FloatTy, dims, "batch", false);
+
+  auto IH = ctx_.allocate(batch)->getHandle();
+  IH.randomize(1.0, 100.0, mod_.getPRNG());
+
+  auto *R = F_->createBatchedReduceMean("reduce.mean", batch, {2, 3}, true);
+
+  auto *save = F_->createSave("save", R);
+  auto *result = ctx_.allocate(save->getPlaceholder());
+  EE_.compile(CompilationMode::Infer, F_);
+
+  EE_.run(ctx_);
+  auto H = result->getHandle();
+
+  // 1st reduce has i*j*k elements for a 4D (i,j,k,l) input & l is reduced
+  std::vector<float> tmpRes(sizeof(float) * dimAllMul);
+  for (size_t i = 0; i < dims[0]; i++) {
+    for (size_t j = 0; j < dims[1]; j++) {
+      for (size_t k = 0; k < dims[2]; k++) {
+        auto index = i * dims[1] * dims[2] + j * dims[2] + k;
+        float res = 0;
+        for (size_t l = 0; l < dims[3]; l++) {
+          res += IH.at({i, j, k, l});
+        }
+        tmpRes[index] = res / dims[3];
+      }
+    }
+  }
+
+  // 2nd reduce has i*j elements when 'k' is further reduced.
+  std::vector<float> results(sizeof(float) * dimResMul);
+  for (size_t i = 0; i < dims[0]; i++) {
+    for (size_t j = 0; j < dims[1]; j++) {
+      auto index = i * dims[1] + j;
+      float res = 0;
+      for (size_t k = 0; k < dims[2]; k++) {
+        res += tmpRes[dims[2] * index + k];
+      }
+      results[index] = res / dims[2];
+
+      EXPECT_NEAR(H.at({i, j, 0, 0}), results[index], 0.001);
+    }
+  }
+}
+
+TEST_P(InterpAndCPU, batchedReduceMeanUsingAvgPoolQuantized) {
```

Comment:
Please use doxygen comments to comment this method.

Reply:
Also fixed similar mistakes in other test files in this PR.